packages/sdk/src/swap/types.ts

```diff
@@ -77,6 +77,7 @@ interface DepositAddressFields extends SwapStatusResponseCommonFields {
   estimatedDepositChannelExpiryTime: number;
   isDepositChannelExpired: boolean;
   depositChannelOpenedThroughBackend: boolean;
+  witnessSafetyMargin: number | undefined;
```

Comment:
i am not sure if we should add this here. i think the minimum amount is kind of the same type of info so maybe we should handle them the same?

Reply:
minimum amount is on the asset data object if i remember correctly, so this would be on the chain data object?

Reply:
that's right, updated it